ansible/roles/postgresql11/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Create vg.metadb Volume Group
+  lvg:
+    vg: vg.metadb
+    pvs:
+      - /dev/xvdb1
+      - /dev/xvdc1
+      - /dev/xvdd1
+      - /dev/xvde1
+      - /dev/xvdf1
+      - /dev/xvdg1
+      - /dev/xvdh1
```

Comment:
Maybe, since these are about the specific disk setup of the particular ams-pg host, these should go outside of the postgresql11 role?

Reply:
Active and standby DB hosts are pretty critical. I think it's safer to dedicate a role for a specific host (instead of having variables and conditionals that can cause unexpected changes on a host A while we are changing host B).

Reply:
Makes sense, perhaps then the role name should contain the active/standby role in it?